wsdl/ver10/schema/onvif.xsd

```diff
@@ -1552,6 +1551,12 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="Extension" type="tt:PTZStatusFilterOptionsExtension" minOccurs="0"/>
+			<xs:element name="FieldOfView" type="xs:boolean" minOccurs="0">
```

Comment:
Suggest to add this as an attribute here in ptzstatusfilteroptions.
And also add a new attribute in <xs:complexType name="PTZFilter"> that used in metadata configuration.

Reply:
I will look into this, sorry for late response.